libs/logging/src/log_documentation.test.ts

```diff
@@ -26,7 +26,6 @@ describe('test cdf documentation generation', () => {
     expect(structuredData.DeviceModel).toEqual('VxAdmin 1.0');
     expect(structuredData.GeneratedDate).toEqual('2020-07-24T00:00:00.000Z');
     expect(structuredData.EventTypeDescription).toHaveLength(5);
-    expect(structuredData.EventIdDescription).toHaveLength(43);
```

Comment:
These assertions are annoying and useless because we just change them when they fail.